src/route-utils.js

```diff
@@ -138,6 +138,21 @@ export function parseRoute({params, routes, location}) {
 class FilterState {
   constructor(state) {
     this.state = state;
+    if (this.state.repoInfos.length === 0) {
+      const repositories = process.env.REPOSITORIES || 'coala';
+      const repoOwner = 'coala'; //default repoOwner
+
+      // extracting repo names
+      let repoNames = repositories.substring(repositories.indexOf(":") + 1).split("|");
```

Comment:
You are using a single quote in another lines but using double quotes here. Be consistent. Use single quote instead.